### PR TITLE
Bug/1121 deprecate allowancefields

### LINKED ISSUE #1121: accountInfo has hbarAllowances which aren't present in the proto definition
### Description

Getting info for an account which has given/received a hbar/token/nft allowance and checking the corresponding properties of the AccountInfo response returns empty arrays.

The protobuf definition for AccountInfo doesn't include these values. The properties should be removed from the corresponding javascript class.

https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_get_info.proto

### Steps to reproduce

Query account info for 0.0.34747646 and 0.0.34747647 on testnet, hbarAllowances is []
Mirror confirms there are indeed allowances https://testnet.mirrornode.hedera.com/api/v1/accounts/0.0.34747575/allowances/crypto?limit=5&order=asc

### Additional context

_No response_

### Hedera network

testnet

### Version

v2.14.0.beta3

### Operating system

_No response_

### PR DESCRIPTION
**Pull request by keirabee**
**Description**:
One or two line summary of what this PR does and why it is needed, followed by a list
of changes in imperative, present tense for use in the commit message or changelog. Example:

This PR adds deprecation comments to AccountInfo to let people know that hbarallowance, tokenallowance and nftallowance are presently deprecated as they cannot presently be removed

**Related issue(s)**:

Fixes #1121 

**Notes for reviewer**:

**Checklist**

- [ X] Documented (Code comments, README, etc.)
- [ X] Tested (unit, integration, etc.)

### CODE REVIEW

File: src/account/AccountInfo.js [hunk 1]
```diff
@@ -31,8 +31,11 @@ import PublicKey from "../PublicKey.js";
 import LedgerId from "../LedgerId.js";
 
 /**
+ * @deprecated - no longer supported
  * @typedef {import("./HbarAllowance.js").default} HbarAllowance
+ * @deprecated - no longer supported
  * @typedef {import("./TokenAllowance.js").default} TokenAllowance
+ * @deprecated - no longer supported
```
Comment (danielakhterov) [inline src/account/AccountInfo.js:38]: `@deprecated` should only be added above the actual fields

File: src/account/AccountInfo.js [hunk 2]
```diff
@@ -202,11 +208,17 @@ export default class AccountInfo {
...
+         */
         this.hbarAllowances = props.hbarAllowances;
-
+        /*
+         *@deprecated - no longer supported
+         */
         this.tokenAllowances = props.tokenAllowances;
-
+        /*
+         *@deprecated - no longer supported
```
Comment (danielakhterov) [inline src/account/AccountInfo.js:220]: Add a space after the star :)

**Review by danielakhterov — CHANGES_REQUESTED** [review]

File: src/account/AccountInfo.js (hunk 2 above)
Comment (keirabee) [reply src/account/AccountInfo.js]: XD

File: src/account/AccountInfo.js (hunk 1 above)
Comment (keirabee) [reply src/account/AccountInfo.js]: updated

File: src/account/AccountInfo.js (hunk 2 above)
Comment (keirabee) [reply src/account/AccountInfo.js]: updated.

File: src/account/AccountInfo.js [hunk 3]
```diff
@@ -301,9 +307,17 @@ export default class AccountInfo {
                 info.ledgerId != null
                     ? LedgerId.fromBytes(info.ledgerId)
                     : null,
-
+            /*
+             * @deprecated - no longer supported
```
Comment (danielakhterov) [inline src/account/AccountInfo.js:311]: I don't think these need to be marked, heh.

**Review by danielakhterov — CHANGES_REQUESTED** [review]

File: src/account/AccountInfo.js (hunk 3 above)
Comment (keirabee) [reply src/account/AccountInfo.js:311]: ok, updated:)

**Review by danielakhterov — APPROVED** [review]

**Pull request merged**